recipes/sphinx_parser/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "sphinx_parser" %}
+{% set version = "0.0.1" %}
+{% set python_min = "3.9" %}
```

Comment:
```suggestion
```

Reply:
I'm not sure if I understand the intention behind this suggestion. I shouldn't set the min version for python? Should I manually set the python version below?